web/styles/left_sidebar.css

```diff
@@ -390,7 +390,7 @@
         background-color: var(--color-background-sidebar-action-heading-hover);
     }
 
-    &.active-filter,
+    &.top-left-active-filter,
```

Comment:
Similarly, this seems wrong; I think you want BOTH `active-filter` and `top-left-active-filter` to have these styles.